rfp-proposal/on-chain-quadratic-funding.md

```diff
@@ -0,0 +1,68 @@
+
+Key-problems to solve as part of the grant:
+1. **Anti-collusion** 
+2. **Sybil resistance**
+3. **User-friendly UI**
+ 
+
+## Deliverables :nut_and_bolt:
+
+The milestones below are just an initial draft. The milestones can be structured completely differently and the implementation can also leverage other tools and infrastructure as long as the overall goal of the RFP is met.  
+
+### Milestone 1 - Anti-collusion
+
+| Number | Deliverable | Specification | 
+| ------------- | ------------- | ------------- |
+| 0a. | License | Apache 2.0 / MIT / Unlicense |
+| 0b. | Documentation | Inline documentation of the code and a basic tutorial that explains how a developer can use the project |
+| 0c. | Testing Guide | The code will have unit-test coverage (min. 70%) to ensure functionality and robustness. In the guide we will describe how to run these tests | 
+| 0d. | Article/Tutorial | Article or tutorial that explains the work done as part of the grant. 
+| 1. | Anti-collusion | Implement a mechanism to prevent bribery and collusion, leveraging encrypting votes (ZKPs) potentially via [Minimum Anti-Collusion Infrastructure (MACI)](https://ethresear.ch/t/minimal-anti-collusion-infrastructure/5413) |  
+| 2.  | Voting Example | Integrate a basic voting example that lets you test the mechanism | 
+
+
+### Milestone 2 - Sybil resistance
+
+| Number | Deliverable | Specification | 
+| ------------- | ------------- | ------------- |
+| 0a. | License | Apache 2.0 / MIT / Unlicense |
+| 0b. | Documentation | Inline documentation of the code and a basic tutorial that explains how a developer can use the project|
+| 0c. | Testing Guide | The code will have unit-test coverage (min. 70%) to ensure functionality and robustness. In the guide we will describe how to run these tests | 
+| 0d. | Article/Tutorial | Article or tutorial that explains the work done as part of the grant. 
+| 1. | Sybil resistance | Mechanism to prevent sybil attacks, either implemented as its own substrate pallet or possible integration of the existing on-chain identity on Kusama/Polkadot provided by specified registrar  |  
+| 2.  | UI | UI for verification of identities, including a polkadot.js extension support | 
+
+### Milestone 3 - CLR
+
+| Number | Deliverable | Specification | 
+| ------------- | ------------- | ------------- |
+| 0a. | License | Apache 2.0 / MIT / Unlicense |
+| 0b. | Documentation | Inline documentation of the code and a basic tutorial that explains how a developer can use the project |
+| 0c. | Testing Guide | The code will have unit-test coverage to ensure functionality and robustness. In the guide we will describe how to run these tests | 
+| 0d. | Article/Tutorial | Article or tutorial that explains the work done as part of the grant. 
+| 1. | CLR pallet(s) | Implement the necessary substrate pallets, potentially containing a Token Curated Registry (TCR) and DAO |  
```

Comment:
> Token Curated Registry (TCR) and DAO

What for?

Reply:
to allow anyone to permissionlessly register eligible recipients each round. I will integrate it.